src/styles/index.scss

```diff
@@ -1,3 +1,162 @@
+@import './variables';
+
+// Міксин для центрованих елементів
+@mixin flex-center($width, $height) {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $width;
+  height: $height;
+}
+
+@mixin star-position {
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+// Міксин для стилю тексту
+@mixin text-style($font-size, $font-weight, $color) {
+  font-size: $font-size;
+  font-weight: $font-weight;
+  color: $color;
+}
+
+html {
+  font-family: $font-family;
+}
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the base font size should be set in the `html` selector using `px` units. Make sure that the `$font-family` variable is defined with a fallback system font, e.g., `Helvetica, Arial, sans-serif`.